rpm/qtdeclarative.spec

```diff
@@ -289,6 +311,17 @@ cp lib/libQt5QmlDevTools.a %{buildroot}/%{_libdir}
 %fdupes %{buildroot}/%{_includedir}
 
 
+# Copy docs
+%if 0%{?qt5_release_build}
+mkdir -p %{buildroot}/%{_docdir}/qt5
+cp -R doc/qtqml %{buildroot}/%{_docdir}/qt5
+cp -R doc/qtqml.qch %{buildroot}/%{_docdir}/qt5
+cp -R doc/qtquick %{buildroot}/%{_docdir}/qt5
+cp -R doc/qtquick.qch %{buildroot}/%{_docdir}/qt5
+%else
```

Comment:
Sounds good, I've updated the patch to remove the %else.

@iamer, with a local build, the subpackage builds fine locally when the macro is not set. The subpackage contains the single (empty) subdirectory, so perhaps that's preventing the build from failing.